LanguageFeatures/Augmentation-libraries/augmenting_types_A05_t04_lib.dart

```diff
@@ -28,3 +28,51 @@ augment class C2<T extends A> {
 augment class C2<T extends AAlias> {
   String name2() => 'C2<$T>';
 }
+
+augment mixin M1<T extends A> {
+  String get name1 => 'M1<$T>';
+}
+augment mixin M1<T extends AAlias> {
+  String name2() => 'M1<$T>';
+}
+augment mixin M2<T extends A> {
+  String get name1 => 'M2<$T>';
+}
+augment mixin M2<T extends AAlias> {
+  String name2() => 'M2<$T>';
+}
+
+augment enum E1<T extends A> {
+  augment e1<B>(),
+  augment e2<AAlias>;
```

Comment:
I think this needs to be
```suggestion
  augment e2<AAlias>();
```